tests/infrastructure/golden_images/test_common_templates_data_volumes.py

```diff
@@ -154,25 +155,20 @@ def test_vm_with_existing_dv(data_volume_scope_function, vm_from_template_with_e
                 "dv_size": FEDORA_LATEST["dv_size"],
             },
             {
-                "updated_storage_class_params": {
-                    "storage_class": StorageClassNames.CEPH_RBD_VIRTUALIZATION,
-                    "access_mode": DataVolume.AccessMode.RWX,
-                    "volume_mode": DataVolume.VolumeMode.BLOCK,
-                },
+                "use_default_storage_class": False,
```

Comment:
`use_default_storage_class` should be renamed to something like `use_ocs_storage_class` to reflect the usage in the code